Modix.Bot/Modules/EmojiStatsModule.cs

```diff
@@ -91,8 +91,8 @@ await ReplyAsync(embed: new EmbedBuilder()
             var guildStats = await _emojiRepository.GetGuildStatsAsync(guildId);
 
             var emojiFormatted = ((SocketSelfUser)Context.Client.CurrentUser).CanAccessEmoji(ephemeralEmoji)
-                    ? Format.Url(ephemeralEmoji.ToString(), EmojiUtilities.GetUrl(ephemeralEmoji.ToString()))
-                    : Format.Url("❔", ephemeralEmoji.Url);
+                    ? ephemeralEmoji.ToString()
```

Comment:
Just double-checking, is this ternary necessary?

Reply:
How else would you do it?

Reply:
Assuming that `GetGuildStatsAsync()` does what I think it does, wouldn't the results be guild-specific, and as such we can ensure that the bot can see them all? Or is there a scenario where some guild-emotes can't be accessed by the bot?

Reply:
This method is used by both the guild version and the all version

Reply:
OH right, just reading the function name made me think it was guild-specific emotes, my bad :)